src/app/dashboard/campaign-card/campaign-flight-dates.pipe.ts

```diff
@@ -27,8 +28,9 @@ export class CampaignFlightDatesPipe implements PipeTransform {
 
   transform(flights: Flight[]): string {
     const dates = this.flightDates(flights);
+    const dateFormat = 'M/D';
```

Comment:
The spec was expecting these dates to not be zero padded (from original design.)